src/nunit-gui/Views/SettingsPage.cs

```diff
@@ -56,16 +51,15 @@ public SettingsPage(string key, SettingsModel settings)
 
         #region Properties
 
+        private bool settingsLoaded;
+
         protected SettingsModel Settings { get; private set; }
 
         protected IMessageDisplay MessageDisplay { get; private set; }
 
         public string Key { get; private set; }
 
-        public bool SettingsLoaded
-        {
-            get { return true; }// _settings != null; }
-        }
+        public bool SettingsLoaded => settingsLoaded;
```

Comment:
I've changed `SettingsLoaded ` into an autoproperty. If `_settings_` could be `null` then I think that we will get NRE in `SettingsDialog`.